Doc/library/datetime.rst

```diff
@@ -570,23 +569,23 @@ Instance attributes (read-only):
 
 .. attribute:: date.month
 
-   Between 1 and 12 inclusive.
+   Between ``1`` and ``12`` inclusive.
 
 
 .. attribute:: date.day
 
-   Between 1 and the number of days in the given month of the given year.
+   Between ``1`` and the number of days in the given month of the given year.
```

Comment:
Not a code.

Reply:
The change has been reverted; marking as resolved.